### PR TITLE
Add script to install dependencies for installation using deb package

### PR DESCRIPTION
**Pull request by pulkitvyas08**

### CODE REVIEW

File: dpkg_requirements.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+DEPS = "python3-pip python3-opencv python3-setuptools python-execnet cmake libatlas-base-dev build-essential"
+apt-get update
+apt-get install -y $DEPS
```
Comment (rushabh-v) [inline dpkg_requirements.sh]: Remove `apt-get update`.
remove the variable and directly put it after `sudo apt install`.

And make it `sudo apt install` (use `sudo` and remove `-get`).

Also, add remaining deps (face-recognition and terminaltables).

**Review by rushabh-v — CHANGES_REQUESTED** [review]

**Comment by rushabh-v** [comment]
The problem with running this script in the installation workflow is that you can't run a `dpkg` command inside another. (facerec will be installed using `dpkg -i` and `apt install` also runs `dpkg` behind the scenes.) So, I think we can just add documentation for installation using deb package and we can put the command to install the dependencies there. (in that case we wouldn't need this script and we just need to specify the `sudo apt` deps there (the pip ones will be installed with the deb package)).

You can suggest a better idea if you have any.

Otherwise, you can start adding documentation for installation using the deb package and remove the shell script (Sorry for the chaos).
Link to the deb package: https://github.com/rushabh-v/linux_face_unlock/releases/download/v1.1/facerec_1.1_all.deb
Also, put a note that the deb package won't receive updates automatically. The PPA method is more preferable.

**Comment by pulkitvyas08** [comment]
I couldn't find a better way so I guess adding documentation would do fine for now. Should I add the documentation in the README file itself or make a separate markdown file?

**Comment by rushabh-v** [comment]
Yes, append it in the readme itself. Thanks!

File: README.md [hunk 2]
```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
 source ~/.bashrc
 ```
 
+### Using .deb package
```
Comment (rushabh-v) [inline README.md]:
```suggestion
### Using deb package
```

File: README.md [hunk 3]
```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
...
+
+#### 1. Download the latest release of Facerec from the releases section of the repository
```
Comment (rushabh-v) [inline README.md]: ` wget https://github.com/rushabh-v/linux_face_unlock/releases/download/v1.1/facerec_1.1_all.deb` to download it.

File: README.md [hunk 4]
```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
...
+
+#### 2. Install the .deb package on your machine using the command
+```
+sudo dpkg -i <name of the .deb file>
```
Comment (rushabh-v) [inline README.md]:
```suggestion
sudo dpkg -i facerec_1.1_all.deb
```

File: README.md [hunk 5]
```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
...
+```
+
+### 3. Install All the dependencies
+```
+sudo apt install -y python3-pip python3-opencv python3-setuptools python-execnet cmake libatlas-base-dev build-essential
```
Comment (rushabh-v) [inline README.md]: put this as step 2 Because we require dependencies before installation of the deb package.

**Review by rushabh-v — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -24,6 +26,28 @@ sudo apt install facerec
 source ~/.bashrc
 ```
 
+### Using deb package
+
+#### 1. Download the deb package of Facerec
```
Comment (rushabh-v) [inline README.md]:
```suggestion
#### 1. Download the deb package
```

File: README.md [hunk 7]
```diff
@@ -24,6 +26,28 @@ sudo apt install facerec
...
+```
+wget https://github.com/rushabh-v/linux_face_unlock/releases/download/v1.1/facerec_1.1_all.deb
+```
+
+#### 2. Install All the dependencies
```
Comment (rushabh-v) [inline README.md]:
```suggestion
#### 2. Install all the dependencies
```

File: README.md [hunk 8]
```diff
@@ -24,6 +26,28 @@ sudo apt install facerec
...
+```
+sudo apt install -y python3-pip python3-opencv python3-setuptools python-execnet cmake libatlas-base-dev build-essential
+```
+
+#### 3. Install the .deb package on your machine using the command
```
Comment (rushabh-v) [inline README.md]:
```suggestion
#### 3. Install the deb package using
```

**Review by rushabh-v — CHANGES_REQUESTED** [review]

**Comment by rushabh-v** [comment]
> Also, put a note that the deb package won't receive updates automatically. The PPA method is more preferable.

also, add this note.

**Review by rushabh-v — APPROVED** [review]

**Comment by rushabh-v** [comment]
give @pulkitvyas08 25 points

**Pull request merged**